features/everything.feature

```diff
@@ -1,67 +1,79 @@
 Feature: A game of connect-4
-Scenario: Starting the game and quitting
-   Given starting main.py
-   When the user selects Quit
-   Then the game closes itself
-
-Scenario: Starting the game and choosing 'Play Game'
-   Given starting main.py
-   When the user selects "Play Game"
-   Then the user gets transferred to the next menu
-
-Scenario: Going back to the start menu
-   Given starting main.py
-   When the user is in the gamemode menu
-   When the user selects the <- back button
-   Then the user gets transferred to the start menu
-
-Scenario: Selecting 'Player vs Player'
+#Scenario: Starting the game and quitting
+#   Given starting main.py
+#   When the user selects Quit
+#   Then the game closes itself
+#
+#Scenario: Starting the game and choosing 'Play Game'
+#   Given starting main.py
+#   When the user selects "Play Game"
+#   Then the user gets transferred to the next menu
+#
+#Scenario: Going back to the start menu
+#   Given starting main.py
+#   When the user is in the gamemode menu
+#   When the user selects the <- back button
+#   Then the user gets transferred to the start menu
+#
+#Scenario: Selecting 'Player vs Player'
+#   Given starting main.py
+#   When the user selects "Play Game"
+#   And the user selects "Player vs Player"
+#   Then the selected gamemode starts
+#
+#Scenario: Player's turn is valid
+#   Given starting main.py
+#   When the user selects "Play Game"
+#   And the user selects "Player vs Player"
+#   And the Player selects column <columns>
+#   Then the move is valid
+#
+#Scenario: Player's turn is invalid
+#   Given starting main.py
+#   When there is a game running
+#   And Player makes invalid move
+#   Then a message will appear which sais that the move is invalid
+#   And the game asks for a new input
+#
+#Scenario: Player wins
+#   Given starting main.py
+#   When there is a game running
+#   And a player wins
+#   Then there is a congratulation message
+#   And the user can return back to the main menu
+#
+#Scenario: Draw
+#   Given starting main.py
+#   When there is a game running
+#   And the board is full
+#   Then the message "The game is a draw!" appears
+#   And the user can return back to the main menu
+#
+#Scenario: Quitting during game
+#   Given starting main.py
+#   When there is a game running
+#   And the Quit button is selected
+#   Then the game quits
+#   And the user will be asked if they want to save the game
+#
+#Scenario: Looking at the Rules
+#   Given starting main.py
+#   When the user presses rules
+#   Then the rules appears
+#   And when the user presses enter, he is back to the main menu
+#
+#Scenario: A game vs AI
+#   Given starting main.py
+#   When the user selects Player vs AI
+#   Then Plays a game vs AI and loses
+#
+Scenario: Testing 5 moves
    Given starting main.py
    When the user selects "Play Game"
    And the user selects "Player vs Player"
-   Then the selected gamemode starts
```

Comment:
You delete a lot of finished tests. Pls update your project, look if everything is okay and submit again

Reply:
I deleted nothing. This comes all because we changed the path, that my behave works. I had to copy paste the whole code into my own version because i couldnt connect me to github.

Reply:
Everything is there so it's fine. Not sure if this messes with the commit history or anything but it is fine. Maybe you are on the wrong branch? You have to switch to master in pycharm to update your project and after that create a new branch from master. If this is not your problem, try to solve it please.

Reply:
@FlorianEder is this fine if the code is still there?